docs/sql/aggregates.md

```diff
@@ -38,6 +38,7 @@ The table below shows the available general aggregate functions.
 
 | Function | Description | Example | Alias(es) |
 |:---|:---|:---|:---|
+| `any_value(arg)` |Returns an arbitary _non-null_ value from arg. | `any_value(A)` | - |
```

Comment:
This isn't correct I don't think? It returns the **first** non-null value, not an arbitrary one?